.tekton/pull-request.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       value: 'quay.io/redhat-appstudio/pull-request-builds:release-service-{{ revision }}'
   pipelineRef:
     name: docker-build
-    bundle: quay.io/redhat-appstudio/hacbs-core-service-templates-bundle:latest
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-core-services-docker-build:latest
```

Comment:
What do you think of tying this to a specific version? Not saying it's needed, just something to consider. (Same thing in the push pipeline.)

Reply:
the idea here was to be customer one of the pipelines, so rather fail for us than for customer.

Reply:
Alright, fair point.